twitter/app/Models/User.php

```diff
@@ -86,6 +89,23 @@ public function getLoginUser(): User|null
     {
         return Auth::user(); 
     }
-}
 
+    /**
+     * マイページの情報編集
+     *
+     * @param string $name
+     * @param string $email
+     */
+
+    public function updateUser(
+        Request $request
+    )
```

Comment:
[imo]
引数がひとつなら、改行しない方が見やすいかも！
```suggestion
    public function updateUser(Request $request)
```

Reply:
ついでに返り値の型指定もあると嬉しい！